lib/wickedbase.pm

```diff
@@ -234,6 +234,31 @@ sub get_test_result {
     }
 }
 
+sub get_wicked_logs {
```

Comment:
also we already have `save_and_upload_wicked_log` don't see the point of having it after this one will be merged

Reply:
I think we could also remove `wickedbase::post_fail_hook()`

Reply:
ok

Reply:
> also we already have `save_and_upload_wicked_log` don't see the point of having it after this one will be merged

I think I missed that function, sorry. Will remove it.